src/components/molecules/AdminVenueCard/AdminVenueCard.scss

```diff
@@ -1,5 +1,7 @@
 @import "scss/constants.scss";
 
+$admin-venue-card-button-margin: 5px 0;
```

Comment:
Is there app wide constant for more uniformed margins and paddings? Something to be used instead of `5px` ?

Reply:
There is not. There are some padding/margin values in `/scss/constants.scss` but we've decided to keep global/common things in there and the component specific ones at the top of their own `.scss` file. @0xdevalias @mike-lvov Correct me, If I'm wrong.

Reply:
Could there be something like a base value, a unit of whitespace in constants, then each component use multiples of it for itself? Right now just having 5px or 16px vs 1 * $space, 3 * $space makes it look inconsistent (why 16, not 15, or 5 instead of 4) and not allowing to just use IDE shortcut to list all places one might need to modify in some future update.

Reply:
What's the benefit? IMO, It will add unnecessary complexity and decrease the readability. You should bring up questions like this one in slack. That way everyone can give input on the subject.

Reply:
My bad, commented before checking. There already **is** something like it in `/scss/constants.scss` :

```scss
$spacing--xs: 4px;
$spacing--sm: 8px;
$spacing--md: 12px;
$spacing--lg: 16px;
$spacing--xl: 24px;
$spacing--xxl: 30px;
```

Reply:
This is something we basically need @sparkletown/design to weigh in on. The expectation is we should be able to choose a minimal set of standards spacing and be able to use that in all cases.